pyproject.toml

```diff
@@ -1,25 +1,20 @@
 [project]
 python = "^3.8"
 
-
 [tool.poetry]
-name = "apim-definitions"
+name = "electronic-prescription-service-api"
 version = "0.0.1-alpha"
 description = "Electronic Prescription Service API Definitions"
-
 license = "MIT"
-
 authors = [
```

Comment:
alphabetical ordering or order of importance??

Reply:
Same ordering as https://python-poetry.org/docs/pyproject/, so I kept it